PatternPal/PatternPal.Core/Recognizers/Helper_Classes/BridgeRecognizerAbstractClass.cs

```diff
@@ -23,7 +24,7 @@ public override MethodCheck HasMethodCheck()
     /// <inheritdoc />
     public override ClassCheck HasInterfaceOrAbstractClassWithMethodCheck(MethodCheck methodInImplementation)
     {
-        return AbstractClass(Priority.Knockout,
+        return AbstractClass(Priority.Knockout, "1. Is the Implementation abstract class.",
```

Comment:
Dit is een kopje dus hier hoeft het geen zin te zijn of een . bij te zetten. In dit geval dus: "1. Implementation Class"